packages/atomic/src/components/search/generated-answer/atomic-generated-answer-feedback/atomic-generated-answer-feedback-modal.tsx

```diff
@@ -0,0 +1,275 @@
+import {GeneratedAnswer} from '@coveo/headless';
+import {GeneratedAnswerFeedback} from '@coveo/headless/dist/definitions/features/generated-answer/generated-answer-analytics-actions';
```

Comment:
This type of import is not correct, and will break.

The interface needs to be exported properly from Headless entry point if it's necessary for implementers to reference it.